.github/workflows/dispatch-workflow.yml

```diff
@@ -14,11 +14,18 @@ on:
 jobs:
   repositories:
     runs-on: ubuntu-latest
-    
+
     outputs:
       matrix: ${{ steps.search-repos.outputs.matrix }}
 
     steps:
+      - name: Check if user has write access
+        uses: lannonbr/repo-permission-check-action@2.0.0
+        with:
+          permission: admin
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We should probably have a fork of the action in use or copy the script over here as otherwise if the 3rdparty action gets compromised it could leak the tokens of org admins to them.

Reply:
This token is the default workflow one. It doesn't contain sensitive data :)

Nonetheless, this doesn't seem to work, I'm considering securing with `if: contains('["skjnldsv"]', github.actor)` for now

Reply:
Ah nice, wasn't aware of that